gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -24,54 +24,60 @@
  * 
  * @author mzechner */
 public class Color {
-	public static final Color WHITE = new Color(1, 1, 1, 1);
-	public static final Color LIGHT_GRAY = new Color(0xbfbfbfff);
-	public static final Color GRAY = new Color(0x7f7f7fff);
-	public static final Color DARK_GRAY = new Color(0x3f3f3fff);
-	public static final Color BLACK = new Color(0, 0, 0, 1);
+	public static final Color WHITE = new ImmutableColor(1, 1, 1, 1);
```

Comment:
All Color constants should have the class `ImmutableColor` or be documented as immutable, otherwise users won't have any indication that they throw `Exception`s when modified.